index.js

```diff
@@ -99,6 +99,9 @@ export async function setupRuby(options = {}) {
   if (inputs['bundler-cache'] === 'true') {
     await common.time('bundle install', async () =>
       bundler.bundleInstall(gemfile, lockFile, platform, engine, version, bundlerVersion, inputs['cache-version']))
+
+    await common.measure(`Print ${lockFile}`, async () =>
```

Comment:
```suggestion
    await core.group`Print ${lockFile}`, async () =>
```
So there is no `Took   0.01 seconds`